data/json/furniture.json

```diff
@@ -746,6 +746,7 @@
     "move_cost_mod": 1,
     "coverage": 40,
     "comfort": 1,
+    "light_emitted": 20,
```

Comment:
I think you may have put this in the wrong spot, this is now a glowing chair.

Reply:
Whoops! I forgot to remove that when I was testing, thanks.